po/pt_BR.po

```diff
@@ -103,11 +103,11 @@ msgstr "Título:"
 #: src/Dialogs/NewAlarmDialog.vala:148
 #, fuzzy
 msgid "Time:"
-msgstr "Cronômetro"
+msgstr "Hora"
```

Comment:
```suggestion
msgstr "Hora:"
```

The original string in English has a colon at the end of it but it's missing in your translation. It's OK if this is intentional though.